src/models/SqlOutputContentProvider.ts

```diff
@@ -411,13 +411,13 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
     public provideTextDocumentContent(uri: vscode.Uri): string {
         // URI needs to be encoded as a component for proper inclusion in a url
         let encodedUri = encodeURIComponent(uri.toString());
-
-        // return dummy html content that redirects to 'http://localhost:<port>' after the page loads
+        let timeNow = new Date().getTime();
```

Comment:
Might be good to add a comment referencing a github issue to know why this was done in the future when everyone has forgotten about this problem.